sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2800,8 +2800,6 @@ object functions {
    * @group datetime_funcs
    * @since 2.0.0
    */
-  @Experimental
-  @InterfaceStability.Evolving
```

Comment:
Yes, I did.  This has been out since 2.0, so I don't think we can change it at this point.